packages/atomic/src/components/search/result-template-components/quickview-iframe/quickview-iframe.ts

```diff
@@ -44,62 +47,86 @@ const warnAboutLimitedUsageQuickview = (logger?: SearchEngine['logger']) => {
   );
 };
 
-/**
- * @deprecated should only be used for Stencil components.
- */
-export const QuickviewIframe: FunctionalComponent<{
+export interface QuickviewIframeProps {
   title: string;
   content?: string;
   onSetIframeRef: (ref: HTMLIFrameElement) => void;
   uniqueIdentifier?: string;
   sandbox?: string;
   src?: string;
   logger?: SearchEngine['logger'];
-}> = ({title, onSetIframeRef, uniqueIdentifier, content, sandbox, src, logger}) => {
+  ref?: RefOrCallback;
+}
+
+export const renderQuickviewIframe: FunctionalComponent<
```

Comment:
This diff is best reviewed by ignoring whitespace
<img width="494" height="302" alt="image" src="https://github.com/user-attachments/assets/4b7fd0a4-a032-4809-aacf-4c8f004fa2c3" />